scripts/release/e2e_test.py

```diff
@@ -82,3 +76,23 @@
     if failures:
         print("One or more tests failed")
         sys.exit(1)
+
+
+if __name__ == '__main__':
+    # Handle command-line args
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input-dir", required=True, type=Path, help="dir path of tests folder")
+    parser.add_argument("-s", "--subscription", required=True, type=str, help="Subscription ID")
+    parser.add_argument("-r", "--resource-group", required=True, type=str, help="Resource group name")
+    parser.add_argument("-w", "--workspace-name", required=True, type=str, help="Workspace name")
+    parser.add_argument("-c", "--coverage-report", required=False, type=Path, help="Path of coverage report yaml")
+    parser.add_argument("-v", "--version-suffix", required=False, type=str,
+                        help="version suffix which will be used to identify the asset id in tests")
+    args = parser.parse_args()
+    tests_dir = args.input_dir
+    subscription_id = args.subscription
+    resource_group = args.resource_group
+    workspace = args.workspace_name
+    coverage_report = args.coverage_report
+    version_suffix = args.version_suffix
+    e2e_test(tests_dir, subscription_id, resource_group, workspace, coverage_report, version_suffix)
```

Comment:
These don't really do anything for the script. Please consider just using the args directly in the call to `e2e_test`:
```suggestion
    e2e_test(args.input_dir, args.subscription, args.resource_group, args.workspace_name,
             args.coverage_report, args.version_suffix)
```